packages/filecoin-address/src/index.ts

```diff
@@ -44,11 +51,11 @@ export class Address {
   }
 
   protocol(): Protocol {
-    return this._protocol
+    return this.bytes[0]
   }
 
   payload(): Uint8Array {
-    return this.str.slice(1, this.str.length)
+    return this.bytes.slice(1)
   }
```

Comment:
@Schwartz10 I feel like this should be normal getters instead of function calls, but not sure if that would be a problem with backward compatibility to change it now.

Reply:
Yeah i would agree, think we could add those, but i would keep these here so we don't break things backwards if we dont need to